addons/web/static/src/core/position_hook.js

```diff
@@ -307,12 +307,17 @@ const POSITION_BUS = Symbol("position-bus");
 export function usePosition(target, options) {
     const popperRef = useRef(options?.popper || DEFAULTS.popper);
     const getTarget = typeof target === "function" ? target : () => target;
+    let moved = false;
     const update = () => {
         const targetEl = getTarget();
         const popperEl = popperRef.el;
         if (!targetEl || !popperEl || !targetEl.ownerDocument.contains(targetEl)) {
             return;
         }
+        if (options.noReposition && moved) {
```

Comment:
Is this because we need to call it a single time?

Not very clear, would it be possible to configure it so it does not call the function at all when it does not need to?

(If we keep this, it should be at the top of the method anyway, right?)

Reply:
> Is this because we need to call it a single time?

Yes

> Not very clear, would it be possible to configure it so it does not call the function at all when it does not need to?

not really, we need to react to the useEffect callback, to position it after the first DOM update

> If we keep this, it should be at the top of the method anyway, right?

No, otherwise if the element is not yet in the DOM, the flag will be set but the element won't be positionned